apps/sms/test/marionette/lib/views/conversation/view.js

```diff
@@ -171,6 +171,25 @@ ConversationView.prototype = {
     );
   },
 
+  enterEditMode: function() {
+    this.showOptions();
+    this.menuAccessors.selectAppMenuOption('Select Messages');
+  },
+
+  exitEditMode: function() {
+    this.back();
+  },
+
+  isInEditMode: function() {
```

Comment:
Well I'm getting `messageNode.displayed()` to be true for every `messageNode`. So it isn't executing `scrollUp()` :disappointed: